UWPHook/GamesWindow.xaml.cs

```diff
@@ -158,6 +250,9 @@ private void BwrSave_DoWork(object sender, DoWorkEventArgs e)
                         {
                             var exePath = @"""" + System.Reflection.Assembly.GetExecutingAssembly().Location + @"""";
                             var exeDir = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+                            List<Task> gridImagesDownloadTasks = new List<Task>();
```

Comment:
So, while the grid images are stored on a user basis, both the CRC and the images themselves are the same for any given user using Steam in the same computer.

We are re-downloading images and re-calculating the CRC for any given game * the number of Steam users detected in the system.

We could improve this a bit by moving a bit of the decision and downloading here to the beginning of the ExportGames task, so that we only download and calculate these once, we may store them in a temp folder for a moment, then copy the images over to each user.

Maybe the crc calculation isn't too much of an overhead, but at least the image download would be cool to make a bit less redundant, since many users don't have fast internet and this may take a while.

What do you think?

Reply:
I agree. I didn't notice this behaviour before. I'll work on this.

Reply:
Made the changes on [22577c8](https://github.com/BrianLima/UWPHook/pull/50/commits/22577c84ae54dec1497850bfe2619d2386b48dc2)

I did as you suggested: download all images to a temp folder, then after the users loop, copy the images to each user grid directory.

It's much faster now!